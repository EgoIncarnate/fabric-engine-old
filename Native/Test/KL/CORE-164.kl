function entry( )
{
	switch( 0 )
	{
	case 0:
		break;
	case 1:
		break;
	var Integer blah[];
	
	}
}
