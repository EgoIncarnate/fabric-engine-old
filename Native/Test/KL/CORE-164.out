9:14: error: code is unreachable
